pixeltable/catalog/table_version.py

```diff
@@ -82,16 +88,25 @@ class TableVersion:
 
     Only TableVersion and Catalog interact directly with stored metadata. Everything else needs to go through these
     two classes.
+
+    TableVersions come in three "flavors" depending on the `effective_version` and `anchor_tbl_id` settings:
+    - if both are None, it's a live table that tracks the latest version
+    - if `effective_version` is defined, it's a snapshot of a specific version
+    - if `anchor_tbl_id` is defined, it's a replica table that is "anchored" to the given table, in the following
```

Comment:
This needs more commentary. See comment for path on line 110, which needs to be expanded.

Reply:
Added comment.